scc/provider/type_subaccount.go

```diff
@@ -281,6 +283,7 @@ func SubaccountResourceValueFrom(ctx context.Context, plan SubaccountConfig, val
 		CloudUser:     plan.CloudUser,
 		CloudPassword: plan.CloudPassword,
 		Tunnel:        tunnel,
+		Connected:     types.BoolValue(value.Tunnel.State == "Connected"),
```

Comment:
As mentioned in the comment above, this value should be optional and defaulted and not be derived.